tests/data_model/test_filters.py

```diff
@@ -1,5 +1,10 @@
+import json
+
 import pytest
 from wikidatarefisland.data_model import StatementFilters
+from wikidatarefisland.data_model.filters import ItemFilters
+from . import mock_data
```

Comment:
PEP8 sorting: First built-in libraries (like json), one empty line, then third party libraries (like request), another empty line, then the other parts your own library. `isort` will handle this automatically for you.

Reply:
cool; I ran isort

Reply:
Thanks. It's really awesome :D